### ISSUE TITLE
Download error:http2: server sent GOAWAY and closed the connection

### ISSUE THREAD

**Issue by aslender** [ncbi/datasets#50]
Hello,

I'm trying to download some genomes for taxons 103695 and 64176. I got this to work previously but for some reason am getting the following error right at the end of the download

Error: Download error: http2: server sent GOAWAY and closed the connection; LastStreamID=3, ErrCode=NO_ERROR, debug="timeout"

I am running the program with the following script

./datasets download genome taxon 64176 --filename podarcis.zip

Help is greatly appreciated.

**Comment by BradHolmes** [comment]
Hi,

I'm sorry to hear you are having trouble downloading data.  This is an unusual error message, and our application logs indicate that the download should have been successful.  Was the file `podarcis.zip` created, and if so, was it a valid zip archive?

I reran you command just now, and it worked.  A network glitch could have happened, so it may be worth trying again.  If the problem persists, please let us know, as we would like to investigate that further.

Thanks,
Brad Holmes
Staff Scientist
NCBI / NLM / NIH

**Comment by aslender** [comment]
Hi Brad,

Thanks for your message. Yes the file podarcis.zip was created. When I tried to unzip it, there was an error saying that it wasn’t a valid zip file. I have been trying multiple times to do this over the last few weeks and still with the same issue. I did get it to work before this though.

I will try downloading the refseq dataset directly from ncbi.

Please let me know if you need any further info.

Thank you, Amy

From: Brad Holmes ***@***.***>
Reply to: ncbi/datasets ***@***.***>
Date: Wednesday, 30 June 2021 at 8:55 pm
To: ncbi/datasets ***@***.***>
Cc: Amy Slender ***@***.***>, Author ***@***.***>
Subject: Re: [ncbi/datasets] Download error:http2: server sent GOAWAY and closed the connection (#50)


Hi,

I'm sorry to hear you are having trouble downloading data. This is an unusual error message, and our application logs indicate that the download should have been successful. Was the file podarcis.zip created, and if so, was it a valid zip archive?

I reran you command just now, and it worked. A network glitch could have happened, so it may be worth trying again. If the problem persists, please let us know, as we would like to investigate that further.

Thanks,
Brad Holmes
Staff Scientist
NCBI / NLM / NIH

**Comment by BradHolmes** [comment]
Thank you for the additional information.  Sorry there wasn't an easy fix.  

I would recommend trying to [download a dehydrated package](https://www.ncbi.nlm.nih.gov/datasets/docs/how-tos/genomes/large-download/), then retrieving the data for the package as a separate step:

```
datasets download genome taxon 64176 --filename podarcis.zip --dehydrated
unzip podarcis.zip -d podarcis
dist/linux-amd64/datasets rehydrate --directory podarcis
```
